src/Documents.jl

```diff
@@ -10,13 +10,17 @@ Defines [`Document`](@ref) and its supporting types
 module Documents
 
 import ..Documenter:
+    Documenter,
     Anchors,
     Formats,
-    Utilities
+    Utilities,
+    # Writers,                  # Only needed for html keywords deprecation
+    Plugin
 
 using DocStringExtensions
 import Markdown
 using Unicode
+# import .Writers: HTMLWriter    # Only needed for html keywords deprecation
```

Comment:
The commented out things should be removed I guess.

Reply:
Yeah, sorry for the mess...